sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -249,6 +249,24 @@ def test_one_hot_encoder_handle_unknown():
     oh = OneHotEncoder(handle_unknown='42')
     assert_raises(ValueError, oh.fit, X)
 
+    # Test the ignore option, when categories are strings
```

Comment:
can you put this in a separate test? (you can leave the code here, but simply add a `def test_...`)